### ISSUE TITLE
My Code doesn't run with tensorscript :-(

### ISSUE THREAD

**Issue by Ulawun** [repetere/ts-deeplearning#1]
Hello,
Thx for sharing everything.
I can run
$ npm test
without problems :-)
When I start any *.mjs file with
$ node --experimental-modules test.mjs
beginning with
import { MultipleLinearRegression, DeepLearningRegression, } from 'tensorscript';
I get the error:
{ Error: Cannot find module tensorscript
    at module.exports (internal/loader/search.js:14:12)
    at Loader.exports.resolve [as resolver] (internal/loader/ModuleRequest.js:102:11)
    at Loader.resolve (internal/loader/Loader.js:82:40)
    at Loader.getModuleJob (internal/loader/Loader.js:112:40)
    at ModuleWrap.module.link (internal/loader/ModuleJob.js:36:25)
    at link (internal/loader/ModuleJob.js:34:19)
    at <anonymous> code: 'MODULE_NOT_FOUND' }
How can I get my test.mjs running ?
Thanks.

**Comment by yawetse** [comment]
Are you using node 10+?

**Comment by yawetse** [comment]
Also does `$ node --experimental-modules manual/examples/ex_regression-boston.mjs` work for you?

**Comment by yawetse** [comment]
hi @Ulawun are you still having an issue?

**Comment by Ulawun** [comment]
Hi yawetse,
Thanks a lot for having a look at the repo.
I'm using node v8.14.0 .
I'm sorry to say, that now with
$ node --experimental-modules manual/examples/ex_regression-boston.mjs
I get the error:
SyntaxError: The requested module does not provide an export named 'get'
    at ModuleJob._instantiate (internal/loader/ModuleJob.js:85:17)
    at <anonymous>

Merry Xmas,
Ulawun

**Comment by yawetse** [comment]
Hi @Ulawun  check out https://github.com/repetere/tensorscript/issues/2, you can run without using MJS just require the bundled common JS file

Example:
```javascript
const TS = require('tensorscript/bundle/tensorscript.cjs');
const MultipleLinearRegression = TS.MultipleLinearRegression;
const Model = new MultipleLinearRegression({ epoch:100, });
const inputData = [
  [ 0, 0, 0, 1, ],
  [ 0, 1, 0, 0, ],
  [ 0, 0, 0, 1, ]
]; 
const outputData = [
  [ 1, ],
  [ 0, ],
  [ 1, ]
];
const predictionData = [
  [ 0, 1, 0, 0 ],
  [ 0, 0, 0, 1 ],
];

Model.train(inputData,outputData)
  .then(() => Model.predict(predictionData,{probability: false}))
  .then(predictions => {
    console.log({ predictions }) //{ predictions: [ [ 0 ], [ 1 ] ] }
  })
  .catch(err=>console.error(err));
```

**Comment by Ulawun** [comment]
Hi Yawetse,

Thanks a lot for your answer.
With your help I now got tensorscript to run.
After the tech. issues are solved,
I'm wondering about mathematics !

From my understanding MultipleLinearRegression
has nothing to do with training and predicting.
I think it is a kind of function approximation.
Why is in your code MultipleLinearRegression trained ?
I think MultipleLinearRegression has to have input data and then
the coefficients of a polynomial are calculated !
Did I got some mathematics wrong ?
Can I get the coefficients calculated with tensorscript ?

Thanks,
Ulawun
